grapher/controls/CollapsibleList.sampleInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { observer } from "mobx-react"
+import { observable, action } from "mobx"
+
+@observer
+class SampleCheckBox extends React.Component<{ id: number }> {
+    @observable checked: boolean = false
+
+    @action.bound onToggle() {
+        this.checked = !this.checked
+    }
+
+    render() {
+        return (
+            <label className="clickable">
+                <input
+                    type="checkbox"
+                    checked={this.checked}
+                    onChange={this.onToggle}
+                />
+                {` checkbox ${this.props.id}`}
+            </label>
+        )
+    }
+}
+
+const items: React.ReactElement[] = []
```

Comment:
Nit: `const collapsibleListSampleItems = range(0,12).map(i => <SampleCheckBox key={i} id={i} />`

Reply:
Excellent! Exactly what I was looking for.